CentralManagement/ChocoCCM Command Reference/Remove-CCMStaleDeployment.md

```diff
@@ -0,0 +1,86 @@
+---
+external help file: ChocoCCM-help.xml
+Module Name: ChocoCCM
+online version:
+schema: 2.0.0
+---
+
+# Remove-CCMStaleDeployment
+
+## SYNOPSIS
+Removes stale CCM Deployments
+
+## SYNTAX
+
+```
+Remove-CCMStaleDeployment [-Age] <String> [-WhatIf] [-Confirm] [<CommonParameters>]
+```

```

Comment:
I kinda feel like this should just be a `-Stale` or `-Prune` switch on `Remove-CcmDeployment`. It doesn't make a ton of sense to me as a separate command.